cms/static/sass/elements/_modules.scss

```diff
@@ -902,3 +914,22 @@ $outline-indent-width: $baseline;
     }
   }
 }
+
+// The "actions menu" for subsections, sections, and units on the Studio course outline page
+.outline .actions-list.nav-dd .wrapper-nav-sub {
+
+  @include text-align(left); // Undo the 'text-align: right' inherited from the parent
+
+  z-index: 10;  // stay in front of other components on the page.
+
+  .nav-item {
+    a {
+      // Match styling of ".wrapper-header nav .nav-item a" (dropdowns in Studio header)
+      color: $gray-d1;
+
+      &:hover {
+        color: $uxpl-blue-hover-active;
+      }
+    }
+  }
+}
```

Comment:
Nit: if we decide to change the styling of these menus, we will need to make changes in two files. Would it make sense to share these rules between the outline and unit views?

https://github.com/open-craft/edx-platform/blob/766fdc5/cms/static/sass/elements/_xblocks.scss#L72-L90

Reply:
Frankly, I didn't worry about the UI maintainability too much as the whole thing is slated to be replaced by an MFE this Fall/Winter.